<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>1140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>501</width>
     <height>99</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1121</width>
     <height>1061</height>
    </rect>
   </property>
   <property name="text">
    <string>www</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>521</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中南民族大学停车收费系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>99</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>现在时间：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>462</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手动操作</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tipsButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>1050</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>使用提示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>1050</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>890</y>
     <width>462</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车辆信息浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>700</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="feeButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>费用查询</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
